releases/release-1.32/release-notes/release-notes-draft.json

```diff
@@ -4319,10 +4802,31 @@
     "duplicate": true,
     "is_mapped": true
   },
+  "128219": {
+    "commit": "f8e64e1d6e44035b5fc0f1e3002e6fade0e6857c",
+    "text": "kubelet: Fix - the volume manager didn't check the device mount state in the #     actual state of the world before marking the volume as detached. It may cause #     a pod to be stuck in the Terminating state due to the above issue when it was #     deleted.",
+    "markdown": "Kubelet: Fix - the volume manager didn't check the device mount state in the #     actual state of the world before marking the volume as detached. It may cause #     a pod to be stuck in the Terminating state due to the above issue when it was #     deleted. ([#128219](https://github.com/kubernetes/kubernetes/pull/128219), [@carlory](https://github.com/carlory))",
```

Comment:
```suggestion
    "markdown": "Kubelet: Fixed - the volume manager didn't check the device mount state in the #     actual state of the world before marking the volume as detached. It may cause #     a pod to be stuck in the Terminating state due to the above issue when it was #     deleted. ([#128219](https://github.com/kubernetes/kubernetes/pull/128219), [@carlory](https://github.com/carlory))",
```

Reply:
Same as above, https://github.com/kubernetes/sig-release/pull/2666/files#r1860996934. To fix this we should either create a map file or wait until the final release notes PR.